privacy-configuration/config1_reference.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
config files are pretty unexciting. TLDR: config1 just has all features disabled, config2 has all features enabled but also has a bunch of exceptions, config3 is an edge-case where feature key is missing